sandbox.go

```diff
@@ -263,6 +266,39 @@ func (sb *sandbox) updateGateway(ep *endpoint) error {
 	return nil
 }
 
+func (sb *sandbox) SetKey(basePath string) error {
+	var err error
+	if basePath == "" {
+		sb.Unlock()
+		return fmt.Errorf("invalid sandbox key")
```

Comment:
am planning on removing the REST-API. but still I will address this comment to be consistent with the rest of the code.